src/admin-views/aggregator/status.php

```diff
@@ -90,22 +100,47 @@
 
 		if ( ! $up || is_wp_error( $up ) ) {
 			$indicator = 'bad';
-			$notes = __( 'The Event Aggregator server is not currently responding', 'the-events-calendar' );
+			$text = __( 'Not connected to', 'the-events-calendar' ) . ' ' . $ea_server;
+			$notes = __( 'The server is not currently responding', 'the-events-calendar' );
 		} elseif ( is_object( $up ) && is_object( $up->data ) && isset( $up->data->status ) && 400 <= $up->data->status ) {
 			// this is a rare condition that should never happen
 			// An example case: the route is not defined on the EA server
 			$indicator = 'warning';
-			$notes = __( 'The Event Aggregator service is responding with an error:', 'the-events-calendar' );
+
+			$text = __( 'Not connected to', 'the-events-calendar' ) . ' ' . $ea_server;
+
+			$notes = __( 'The server is responding with an error:', 'the-events-calendar' );
 			$notes .= '<pre>';
 			$notes .= esc_html( $up->message );
 			$notes .= '</pre>';
+		} else {
+			$text = __( 'Connected to', 'the-events-calendar' ) . ' ' . $ea_server;
 		}
 
-		// @todo - eventually this should link to the status page
-		$text = $ea_server;
+		// @todo - eventually link $text to the status page
 		?>
 		<tr>
-			<td class="label"><?php esc_html_e( 'Server URL', 'the-events-calendar' ); ?></td>
+			<td class="label"><?php esc_html_e( 'Server Connection', 'the-events-calendar' ); ?></td>
+			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
+			<td><?php echo esc_html( $text ); ?></td>
+			<td><?php echo $notes; ?></td>
+		</tr>
+		<?php
+		$indicator = 'good';
+		$notes = '&nbsp;';
+
+		// @todo add API request for pingback check
+		if ( defined( 'DISABLE_WP_CRON' ) && true === DISABLE_WP_CRON ) {
+			$indicator = 'warning';
+			$text = __( 'WP Cron not enabled', 'the-events-calendar' );
```

Comment:
`esc_html__()` in this block (and in the matching else block) would be preferred